### PR TITLE
Documented service annotation for AWS ELB SSL policy

### PR DESCRIPTION
**Pull request by micahhausler**
This change is dependent on kubernetes/kubernetes#54507

ping @justinsb

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6261)

### CODE REVIEW

**Comment by chenopis** [comment]
@micahhausler FYI, the Travis CI error has been fixed. Please rebase or update your branch. Thx!

File: docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -561,6 +561,23 @@ In the above example, if the service contained three ports, `80`, `443`, and
...
 
+Beginning in 1.9, services can use [predefined AWS SSL policies](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html)
+for any HTTPS or SSL listeners. To see which policies are available for use, run
+the awscli command:
+
+```bash
+aws elb describe-load-balancer-policies --query 'PolicyDescriptions[].PolicyName'
+```
+
+Any one of those policies can then be specified using the following annotation:
```
Comment (tengqm) [inline docs/concepts/services-networking/service.md]: (trying to be picky ...)

> using the following annotation:

using the "`service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy`" annotation, for example:

Comment (micahhausler) [reply docs/concepts/services-networking/service.md]: No problem! Thanks

**Comment by steveperry-53** [comment]
@micahhausler Can you resolve the merge conflict? Thanks.

**Comment by micahhausler** [comment]
@steveperry-53 This is ready for merge.

**Pull request merged**